### DISCUSSION TITLE
Opening SMPlayer takes 10 minutes

### DISCUSSION THREAD

**Discussion by Kriss2710** [smplayer-dev/smplayer#820] [Q&A] [answered]
Hi,
When I open SMPlayer it takes upto 10 minutes to open.
I uninstalled it, downloaded the latest version and installed it. 
No change.

SMPlayer is the only program I have this kind of problem with.
Is there an option I have clicked and it remained after I uninstalled it ?

Has anybody had the same problem ?

Thanks for helping
Kris

**Comment by smplayer-dev** [comment, marked as answer]
That's weird.
Try removing (or renaming) the smplayer configuration folder.

**Reply by Kriss2710** [reply]
Thanks a lot.
I renamed the .smplayer folder in my C:\Users\myuser folder
Re installed SMPlayer and it works perfectly
Thanks again.
